openfisca_france/parameters/impot_revenu/calcul_revenus_imposables/deductions/abatviag/taux2.yaml

```
@@ -1,12 +1,12 @@
-description: Taux imposable pour les rentes viagères à titre onéreux des personnes âgées de 50 à 59 ans
+description: Fraction imposable de la rente viagère à titre onéreux pour les personnes âgées de 50 à 59 ans
 values:
   1979-07-01:
     value: 0.5
 metadata:
-  last_value_still_valid_on: "2024-07-30"
+  last_value_still_valid_on: "2024-10-21"
   unit: /1
-  short_label: De 50 à 59 ans
+  short_label: Crédirentier de 50 à 59 ans
   reference:
     1979-07-01:
-      title: Art. 158 du Code général des impôts
+      title: Article 158-6 du Code général des impôts
```

```suggestion
      title: Article 158, 6. du Code général des impôts
```

@benoit-cty je l'aurais plus écrit de cette façon, car c'est le 6 de l'article 158 ?